dbms/src/Interpreters/Aggregator.cpp

```diff
@@ -285,18 +285,12 @@ Aggregator::Aggregator(const Params & params_, const String & req_id)
         /// init spiller if needed
         auto header = getHeader(false);
         bool is_convertible_to_two_level = AggregatedDataVariants::isConvertibleToTwoLevel(method_chosen);
-        bool is_input_support_spill = Spiller::supportSpill(header);
-        if (is_convertible_to_two_level && is_input_support_spill)
+        if (is_convertible_to_two_level)
         {
             /// for aggregation, the input block is sorted by bucket number
             /// so it can work with MergingAggregatedMemoryEfficientBlockInputStream
             spiller = std::make_unique<Spiller>(params.spill_config, true, 1, header, log);
         }
```

Comment:
still need to keep the `else` branch here?

Reply:
ok, fixed.